src/os/ObjectStore.h

```diff
@@ -756,9 +756,15 @@ class ObjectStore {
    *
    * @return iterator, null on error
    */
+  struct omap_iter_seek_t {
+    std::string key;
+    bool lower_bound = true;
+    static omap_iter_seek_t def() { return {}; }
```

Comment:
I worry the `default` is a [C++ keyword](https://en.cppreference.com/w/cpp/keyword), so it would need to be written `Default()`. This is the reason behind going with `def()`.

Reply:
Yeah, right, I forgot that default is a keyword <facepalm>.

Reply:
Let's leave `def` then.

Reply:
Rather than def or default, how min_lower_bound?  It's less interesting that it's a default than what the default encodes.  It would make the places where start_from has a default argument self documenting.

Reply:
@rzarzynski ^

Reply:
Switched to `min_lower_bound`.